docs/content/concepts/assets/asset-checks/checking-for-data-freshness.mdx

```diff
@@ -7,13 +7,15 @@ description: "Use freshness checks to identify data assets that are overdue for
 
 A freshness check is a type of [asset check](/concepts/assets/asset-checks) that allows you identify Dagster [assets](/concepts/assets/software-defined-assets) that are overdue for a data refresh.
 
-As freshness checks don't depend on a specific root cause, it makes them helpful in accounting for unknowns. For example, freshness checks can identify stale assets caused by any of the following:
+Freshness checks don't depend on a specific root cause, which makes them helpful in accounting for unknowns. For example, you can use freshness checks to identify stale assets caused by:
 
 - The pipeline hitting an error and failing
 - Runs were never scheduled
 - A backed up run queue
 - Runs are taking longer than expected to complete
 
+For example, let's say your team uses a dashboard as part of their daily stand up. You determine that the assets powering the dashboard should be materialized no later than 7AM to ensure the data is up-to-date. If a freshness check runs at that time and the assets haven't been updated, the check will fail. The freshness check ensures you're notified when data doesn't arrive as expected without needing to know the specific cause.
```

Comment:
"The freshness check ensures you're notified" isn't exactly accurate. Dagster+ users can set up alert policies to send notifications when checks fail, but notifications won't be sent otherwise. Maybe replace with the more vague "ensures you know"?

Also tiny nitpick is that I think a comma between "expected" and "without" could help avoid some potential ambiguity, but feel free to ignore.

Reply:
I hear you on both suggestions! I'll make the changes.